### ISSUE TITLE
Position of Voice name is corrupted

### ISSUE THREAD

**Issue by bwl21** [moinejf/abc2svg#74]
I updated to 1.15.8

We often specify snm for a voice only. This now causes Problems.

```
X:1
F:1_abcsvgvoiceshortnames
T:abcsvg-voice-shortnames
C:
S:
I:linebreak $
M:4/4
L:4/4
Q:1/4=120
K:C
%
%
%%score 1 2
%
V:1 clef=treble snm="voice 1"
c | $
d |
V:2 clef=treble snm="voice 1"
B C |
```

Yields

![screenshot_1097](https://user-images.githubusercontent.com/2022976/35184301-06a84cec-fdf4-11e7-886f-334b0cc5b7fc.jpg)

while in 1.15.5 i get

![screenshot_1098](https://user-images.githubusercontent.com/2022976/35184294-f996123c-fdf3-11e7-8307-6941225331bc.jpg)

**Comment by moinejf** [comment]
Should be fixed. Thanks.

**Comment by bwl21** [comment]
Yes, it is fixed inabc2svg-1.15.8-14-g44ae2d7 (2018-01-22)
